src/app/home/home.component.scss

```diff
@@ -1,9 +1,50 @@
-.x-small small {
-    font-size: 60%;
+$debug-pf-border:0px;
+
+$content-offset: 15px;
+
+$debug-pf-color: purple;
+$debug-pf-row-color:lighten($debug-pf-color,25%);
+$debug-pf-column-color:lighten($debug-pf-color,50%);
+
+
+.container-cards-pf
+{
+  position:relative;
+  border :solid $debug-pf-border $debug-pf-color;
+  width:100%;
+  margin-top:0px;
+  overflow:auto;
+}
+.row
+{
+  padding-top:$content-offset/2;
+  position:relative;
+  border:solid $debug-pf-border $debug-pf-row-color;
 }
 
-.pull-bottom {
-  display: table-cell;
-  vertical-align: bottom;
-  float: none;
+div[class*='col-']
+{
+  position:relative;
+  border:solid $debug-pf-border $debug-pf-column-color;
+  padding-left:$content-offset;
+  padding-right:$content-offset;
+  .card-pf
+  {
+    padding:0px $content-offset;
```

Comment:
The W3 spec on css only says that the units are optional. I'm from the school of thought that they should not be used since 0 equals the same value for all units. 
We are not using 'px' except to define the base for 'em', so I'm not sure they should be 'px' but should be 'em'. I caught it because I saw the use of 'px'. 
All that said, if this is the only thing left I won't hold up the merge for it.